Gemfile

```diff
@@ -41,44 +40,45 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # See miq_expression_spec Date/Time Support examples.
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
```

Comment:
:sparkles: :us: 

It would be worth investigating if other rails apps use american_date cc @matthewd @tenderlove (see above)